### ISSUE TITLE
PEG.js is faster

### ISSUE THREAD

**Issue by indutny** [veged/ometa-js#10]
PEG.js in quite faster than OmetaJS, need to find a way to speedup grammars.

**Comment by hax** [comment]
bug peg.js does not support left recusion ?

**Comment by indutny** [comment]
@hax indeed, but I think ometa may work as fast as peg does now, taking in account that my fork supports "compiler hints" (`@` to tell compiler that rule doesn't contain left recursion).
